inc/search.class.php

```diff
@@ -3272,9 +3412,11 @@ static function addDefaultToView($itemtype, $params) {
     *
     * @param string $itemtype device type
     *
-    * @return string Select string
+    * @return string Select SQL string
+    *
+    * @since 10.0.0 Method is no longer static
    **/
-   static function addDefaultSelect($itemtype) {
+   public function addDefaultSelect($itemtype) {
```

Comment:
This method is used in some plugins (`fusioninventory` for instance), change should be listed in changelog.